### PR TITLE
feat(StatusTag): Add Alpha style

### PR DESCRIPTION
**Pull request by rivka-ungar**
https://monday.monday.com/boards/3532714909/views/113186840/pulses/5454353803

### CODE REVIEW

File: packages/storybook-blocks/src/components/status-tag/status-tag.module.scss [hunk 1]
```diff
@@ -4,7 +4,7 @@
   padding: 0 var(--sb-spacing-small);
   text-transform: capitalize;
 
-  &.beta {
+  &.alpha, &.beta {
```
Comment (YossiSaadi) [inline packages/storybook-blocks/src/components/status-tag/status-tag.module.scss:7]: Do we want it with same style on purpose?

**Review by YossiSaadi — APPROVED** [review]
Seems cool 💪🏻

File: packages/storybook-blocks/src/components/status-tag/status-tag.module.scss (hunk 1 above)
Comment (rivka-ungar) [reply packages/storybook-blocks/src/components/status-tag/status-tag.module.scss:7]: Yep, thats what it said in the task :)

Comment (YossiSaadi) [reply packages/storybook-blocks/src/components/status-tag/status-tag.module.scss:7]: Perfect 👌🏻

**Pull request merged**